SSA/Projects/RISCV64/Base.lean

```diff
@@ -145,151 +145,151 @@ encounters a `sig` it can replace it by its definition.
-  |srai (shamt : BitVec 6) => Ty.bv
-  |.addw => Ty.bv
-  |.subw => Ty.bv
-  |.sllw => Ty.bv
-  |.srlw => Ty.bv
-  |.sraw => Ty.bv
-  |.add => Ty.bv
-  |.slt => Ty.bv
-  |.sltu => Ty.bv
-  |.and => Ty.bv
-  |.or => Ty.bv
-  |.xor => Ty.bv
-  |.sll => Ty.bv
-  |.srl => Ty.bv
-  |.sub => Ty.bv
-  |.sra => Ty.bv
-  |.remw  => Ty.bv
-  |.rem => Ty.bv
-  |.mul => Ty.bv
-  |.mulw => Ty.bv
-  |.div => Ty.bv
-  |.divw => Ty.bv
-  |.divwu => Ty.bv
-  |.addi (imm : BitVec 12) => Ty.bv
-  |.slti (imm : BitVec 12) => Ty.bv
-  |.sltiu (imm : BitVec 12) => Ty.bv
-  |.andi (imm : BitVec 12) => Ty.bv
-  |.ori (imm : BitVec 12) => Ty.bv
-  |.xori (imm : BitVec 12) => Ty.bv
-  |.czero.eqz => Ty.bv
-  |.czero.nez => Ty.bv
-  |.sext.b => Ty.bv
-  |.sext.h => Ty.bv
-  |.zext.h => Ty.bv
-  |.bclr => Ty.bv
-  |.bext => Ty.bv
-  |.binv => Ty.bv
-  |.bset => Ty.bv
-  |.bclri (shamt : BitVec 6) => Ty.bv
-  |.bexti (shamt : BitVec 6) => Ty.bv
-  |.binvi (shamt : BitVec 6) => Ty.bv
-  |.bseti (shamt : BitVec 6) => Ty.bv
-  |.rolw => Ty.bv
-  |.rorw => Ty.bv
-  |.add.uw => Ty.bv
-  |.sh1add.uw => Ty.bv
-  |.sh2add.uw => Ty.bv
-  |.sh3add.uw => Ty.bv
-  |.sh1add => Ty.bv
-  |.sh2add => Ty.bv
-  |.sh3add => Ty.bv
+  | .li _ => Ty.bv
+  | .mulu => Ty.bv
+  | .mulh => Ty.bv
+  | .mulhu => Ty.bv
+  | .mulhsu => Ty.bv
+  | .divu => Ty.bv
+  | .rol => Ty.bv
+  | .ror => Ty.bv
+  | .remwu => Ty.bv
+  | .remu =>  Ty.bv
+  | .addiw (_imm : BitVec 12) => Ty.bv
+  | .lui (_imm : BitVec 20) => Ty.bv
+  | .auipc (_imm : BitVec 20) => Ty.bv
+  | .slliw (_shamt : BitVec 5) => Ty.bv
+  | .srliw (_shamt : BitVec 5) => Ty.bv
+  | .sraiw (_shamt : BitVec 5) => Ty.bv
+  | .slli (_shamt : BitVec 6) => Ty.bv
+  | .srli (_shamt : BitVec 6) => Ty.bv
+  | .srai (_shamt : BitVec 6) => Ty.bv
+  | .addw => Ty.bv
+  | .subw => Ty.bv
+  | .sllw => Ty.bv
+  | .srlw => Ty.bv
+  | .sraw => Ty.bv
+  | .add => Ty.bv
+  | .slt => Ty.bv
+  | .sltu => Ty.bv
+  | .and => Ty.bv
+  | .or => Ty.bv
+  | .xor => Ty.bv
+  | .sll => Ty.bv
+  | .srl => Ty.bv
+  | .sub => Ty.bv
+  | .sra => Ty.bv
+  | .remw  => Ty.bv
+  | .rem => Ty.bv
+  | .mul => Ty.bv
+  | .mulw => Ty.bv
+  | .div => Ty.bv
+  | .divw => Ty.bv
+  | .divwu => Ty.bv
+  | .addi (_imm : BitVec 12) => Ty.bv
+  | .slti (_imm : BitVec 12) => Ty.bv
+  | .sltiu (_imm : BitVec 12) => Ty.bv
+  | .andi (_imm : BitVec 12) => Ty.bv
+  | .ori (_imm : BitVec 12) => Ty.bv
+  | .xori (_imm : BitVec 12) => Ty.bv
+  | .czero.eqz => Ty.bv
+  | .czero.nez => Ty.bv
+  | .sext.b => Ty.bv
+  | .sext.h => Ty.bv
+  | .zext.h => Ty.bv
+  | .bclr => Ty.bv
+  | .bext => Ty.bv
+  | .binv => Ty.bv
+  | .bset => Ty.bv
+  | .bclri (_shamt : BitVec 6) => Ty.bv
+  | .bexti (_shamt : BitVec 6) => Ty.bv
+  | .binvi (_shamt : BitVec 6) => Ty.bv
+  | .bseti (_shamt : BitVec 6) => Ty.bv
+  | .rolw => Ty.bv
+  | .rorw => Ty.bv
+  | .add.uw => Ty.bv
+  | .sh1add.uw => Ty.bv
+  | .sh2add.uw => Ty.bv
+  | .sh3add.uw => Ty.bv
+  | .sh1add => Ty.bv
+  | .sh2add => Ty.bv
+  | .sh3add => Ty.bv
```

Comment:
These formatting changes (and the ones above) seem unrelated, and it is unclear why they were applied. I would suggest to pull those into a separate PR where you document which convention you implement here. 

Some context:

- as discussed previously, mixing changes in PRs typically increases complexity

- formatting changes in a repository work typically best if we use this opportunity to (a) clarify the formatting conventions we aim to establish and (b) ensure that these conventions are consistently applied in a given file.


Especially the latter is important, to ensure we (a) converge to a given set of formatting guidelines and (b) other project members are educated about the formatting guidelines you establish.